database/db.js

```diff
@@ -0,0 +1,55 @@
+const DATABASE = 'kickstarter';
+const USERNAME = 'root';
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize({
+  database: DATABASE,
+  username: USERNAME,
+  password: null,
+  host: 'localhost',
+  dialect: 'mysql',
+  define: {
+    allowNull: false
+  },
+  sync: { force: true }
+});
+
+sequelize
+  .authenticate()
+  .then(() => {
+    console.log('MYSQL connection has been established...');
+  })
+  .catch(err => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+const User = sequelize.define('user', {
+  userId: { type: Sequelize.INTEGER, primaryKey: true },
```

Comment:
In the same vein as my previous comment, if you want to cut a couple lines of code here, you could.  If you don't specify anything, Sequelize automatically defaults to the id column for its primary key. The id column is also automatically created for you if you don't explicitly define it in your model (i.e. you could completely cut out lines 27, 34, & 42, if you wish).
Also, when you are defining a Sequelize model, you do not need to specify the 'type' option. I figure you decided to specify 'type' because of the 'primaryKey' option, but esp. if you decide to not explicitly define the id columns for each table, I feel you don't need to specify 'type'. (just to clarify, I'm advocating for 
projectName: Sequelize.STRING   over 
projectName: { type: Sequelize.STRING } ) .
I'm always for less typing haha.

Reply:
Good point. Thanks :)